packages/react-dom/src/__tests__/DOMPropertyOperations-test.js

```diff
@@ -157,6 +157,44 @@ describe('DOMPropertyOperations', () => {
     });
   });
 
+  it('should warn if custom attributes take too long to stringify', () => {
+    const container = document.createElement('div');
+    const attributeValue = {foo: 'bar'};
+    attributeValue.toString = function() {
+      // finds 2000th prime to waste time
```

Comment:
could you make this something simpler? Like a for loop that just runs many times. We shouldn't have to maintain a prime number generator in our tests :)

Reply:
added for loop